packages/builder/src/pages/builder/app/[application]/design/[screenId]/_components/ComponentList/ComponentTree.svelte

```diff
@@ -109,6 +118,9 @@
         on:dragover={dragover(component, index)}
         on:iconClick={() => toggleNodeOpen(component._id)}
         on:drop={onDrop}
+        hovering={$store.hoverComponentId === component._id}
+        on:mouseover={() => handleMouseover(component._id)}
+        on:mouseout={() => handleMouseout(component._id)}
```

Comment:
Using `mouseover` and `mouseout` here thrashes the frontend store extremely hard (which cripples the page because everything depends on the frontend store). `mouseenter` and `mouseleave` are the events we should use here.

`mouseover` fires when you enter a component or its children, while `mouseenter` only fires for the outermost component, which is the behaviour we want. Same in reverse for `mouseleave` vs `mouseout`.

So we should update these events (and therefore the core nav item) to use `mouseenter` and `mouseleave` instead.

The same applies for the other nav item usages but I won't duplicate this comment.